workspaces/ballerina/ballerina-extension/src/rpc-managers/common/rpc-manager.ts

```
@@ -187,11 +187,15 @@ export class CommonRpcManager implements CommonRPCAPI {
             if (params.commands.length >= 1) {
                 const cmdArgs = params.commands.length > 1 ? params.commands.slice(1) : [];
-                await commands.executeCommand(params.commands[0], ...cmdArgs);
-                resolve({ data: "SUCCESS" });
+                try {
+                    await commands.executeCommand(params.commands[0], ...cmdArgs);
+                    resolve({ data: "SUCCESS" });
+                } catch (error) {
+                    reject(error);
+                }
             }
         });
```

Biome (2.4.7)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)